utils/constants.ts

```diff
@@ -112,147 +112,147 @@ export const chains: ChainProps[] = [
     address: "",
     chain: ChainKey.ARBITRUM,
     decimals: 18,
-    isDefault: true,
+    id: "0x1",
     ticker: "ETH",
   },
   {
     address: "",
     chain: ChainKey.AVALANCHE,
     decimals: 18,
-    isDefault: true,
+    id: "0xa86a",
     ticker: "AVAX",
   },
   {
     address: "",
     chain: ChainKey.BASE,
     decimals: 18,
-    isDefault: true,
+    id: "0x2105",
     ticker: "ETH",
   },
   {
     address: "",
     chain: ChainKey.BITCOIN,
     decimals: 8,
-    isDefault: true,
+    id: "0x1f96",
     ticker: "BTC",
   },
   {
     address: "",
     chain: ChainKey.BITCOINCASH,
     decimals: 8,
-    isDefault: true,
+    id: "0x2710",
     ticker: "BCH",
   },
   {
     address: "",
     chain: ChainKey.BLAST,
     decimals: 18,
-    isDefault: true,
+    id: "0x13e31",
     ticker: "ETH",
   },
   {
     address: "",
     chain: ChainKey.BSCCHAIN,
     decimals: 18,
-    isDefault: true,
+    id: "0x38",
     ticker: "BNB",
   },
   {
     address: "",
     chain: ChainKey.CRONOSCHAIN,
     decimals: 18,
-    isDefault: true,
+    id: "0x19",
     ticker: "CRO",
   },
   {
     address: "",
     chain: ChainKey.DOGECOIN,
     decimals: 8,
-    isDefault: true,
+    id: "0x7d0",
     ticker: "DOGE",
   },
   {
     address: "",
     chain: ChainKey.DYDX,
     decimals: 18,
-    isDefault: true,
+    id: "",
     ticker: "DYDX",
   },
   {
     address: "",
     chain: ChainKey.DASH,
     decimals: 8,
-    isDefault: true,
+    id: "",
     ticker: "DASH",
   },
   {
     address: "",
     chain: ChainKey.ETHEREUM,
     decimals: 18,
-    isDefault: true,
+    id: "0x1",
     ticker: "ETH",
   },
   {
     address: "",
     chain: ChainKey.GAIACHAIN,
     decimals: 6,
-    isDefault: true,
+    id: "",
     ticker: "ATOM",
   },
   {
     address: "",
     chain: ChainKey.KUJIRA,
     decimals: 6,
-    isDefault: true,
+    id: "",
     ticker: "KUJI",
   },
   {
     address: "",
     chain: ChainKey.LITECOIN,
     decimals: 8,
-    isDefault: true,
+    id: "",
     ticker: "LTC",
   },
   {
     address: "",
     chain: ChainKey.MAYACHAIN,
     decimals: 10,
-    isDefault: true,
+    id: "",
```

Comment:
I think vulticonnect is mostly relevant to EVM chains , not thorchain , not maya, not UTXO chains